### ISSUE TITLE
Add unsettled practices and current limitations section

### ISSUE THREAD

**Issue by fjclark** [alchemistry/alchemical-best-practices#151]
# Proposed Change

Add unsettled practices and current limitations section. 

# Proposed Authors and Reviewers

We expect many contributions from the community - please add/ discuss suggestions below. We will summarise and add these to the paper around May.

**Comment by jmichel80** [comment]
Happy to contribute to this section. 
Areas with lack of best practices to highlight
- IDPs
- covalent ligands
- RNA

**Comment by jmichel80** [comment]
I have added a paragraph to highligh the above 3 problem classes

**Comment by fjclark** [comment]
Thank you! @ppxasjsm do you have any suggestions/ could you please review (see added section at the end of the article - I'm unsure why this isn't a tracked change as all changes made by others have been)? 

@davidlmobley do you have any suggestions for other unsettled practices/ current limitations?

Suggestions welcome from anyone else reading this - it would be great to have everyone's thoughts here.

Thanks.

**Comment by davidlmobley** [comment]
Indeed, I would merge these two. I'll re-read these right now and see if I see anything obviously missing.

**Comment by davidlmobley** [comment]
OK, merged the two sections and added a couple sentences pointing back to the charge corrections section, which I consider a "somewhat unsettled" issue.

**Comment by davidlmobley** [comment]
Marking this as closed for now, feel free to reopen if someone wants to add more.
